man/systemd-random-seed.service.xml

```diff
@@ -23,7 +23,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-random-seed.service</filename></para>
-    <para><filename>/usr/lib/systemd/random-seed</filename></para>
+    <para><filename>/var/lib/systemd/random-seed</filename></para>
```

Comment:
oh, i think this is supposed to say:

`/usr/lib/systemd/systemd-random-seed`

i.e. we usually specify to command this is about in the synopsys (systemd-random-seed.service just calls that binary). And `/var/lib/systemd/random-seed` is not the command, but the saved seed

Reply:
Ah, yes, I though it just mentions the file where the seed is stored. Will fix.